app/furniture/marketplace/marketplace.rb

```diff
@@ -30,9 +30,13 @@ def ready_for_shopping?
       products.present? && stripe_api_key? && delivery_areas.present? && stripe_account_connected?
     end
 
+    def stripe_utility
+      space.utilities.find_by!(utility_slug: :stripe).utility
+    end
```

Comment:
Then I can also reuse this in the new overview tile component: https://github.com/zinc-collective/convene/pull/1682/files?diff=split&w=1#diff-49b1666c6a6edea9fcc01d928b068af8cb31fbf259e376289098a3c6d2a9379bR9

Reply:
Nice! I do continue to worry that the way we find the `stripe_utility` is fraught with peril; but that's out of scope.